src/main/java/log_out/ui/LogOutView.java

```diff
@@ -19,12 +19,13 @@ public LogOutView(LogOutController controller){
     public void propertyChange(PropertyChangeEvent evt) {
```

Comment:
Not for this line particular, but the log out view should create a JButton with an action listener to detect being clicked and calling the log out controller